### PR TITLE
Add method to destroy stored captcha phrase and fix possible replay attack vulnerability

### PR DESCRIPTION
**Pull request by freezy-sk**

### CODE REVIEW

File: src/Service/Captcha.php [hunk 1]
```diff
@@ -94,6 +94,19 @@ public function verify($inputPhrase)
             return false;
         }
 
-        return $inputPhrase === $storedPhrase;
+        if ($inputPhrase === $storedPhrase) {
+            $this->destroy();
+            return true;
```
Comment (fabschurt) [inline src/Service/Captcha.php]: Could you please add a newline above `return true;`&nbsp;?

File: src/Service/CaptchaInterface.php [hunk 2]
```diff
@@ -38,4 +38,11 @@ public function generate($phrase = null, $width = null, $height = null);
      * @return bool
      */
     public function verify($inputPhrase);
+
+    /**
+     * Destroy current captcha phrase.
```
Comment (fabschurt) [inline src/Service/CaptchaInterface.php]: > Destroy**s**

(To be consistent with how verbs are used in the other DocBlock summaries.)

**Comment by fabschurt** [comment]
Hey @freezy-sk&nbsp;! Nice to see you again. Thanks a lot for this very useful contribution. I’ve just requested 2 minor changes regarding CS, but otherwise it’s all great, thanks&nbsp;! 👍

**Comment by freezy-sk** [comment]
Hi @fabschurt 
It's all done right now ;)

**Pull request merged**